sdk/instrumentation/google.golang.org/grpc/server.go

```diff
@@ -231,3 +237,14 @@ func WrapStatsHandler(delegate stats.Handler, spanFromContext sdk.SpanFromContex
 		dataCaptureConfig: internalconfig.GetConfig().GetDataCapture(),
 	}
 }
+
+func setSchemeAttributes(ctx context.Context, span sdk.Span) {
+	peer, ok := peer.FromContext(ctx)
+	if ok {
+		if peer.AuthInfo != nil {
```

Comment:
nit: I'd rather resolve the scheme on before hand and only call SetAttribute once.